onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -239,19 +239,20 @@ struct MyCustomKernel {
 };
 
 struct MyCustomOp : Ort::CustomOpBase<MyCustomOp, MyCustomKernel> {
```

Comment:
There are two DLLs, they contains same op schema.  So one of them will get registered into the registry. Then one dll will depends on another, But the unload order is not clear. So they could get crashed at the ending?

Reply:
Not sure if I'm understanding the question correctly. The schema is not provided by the custom ops developer. He/She provides vector<OrtCustomOp*> and we (ORT) construct the schema on the fly for each op in this vector. See [this](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/session/custom_ops.cc#L105). Nothing changes here w.r.t 2 DLLs.